set.mm

```diff
@@ -18227,9 +18227,8 @@ Converse of the inference rule of (universal) generalization ~ ax-gen .
     $( An equality theorem for substitution.  (Contributed by NM, 16-May-1993.)
        Revise ~ df-sb .  (Revised by BJ, 22-Dec-2020.) $)
     sbequ1 $p |- ( x = t -> ( ph -> [ t / x ] ph ) ) $=
-      ( vy weq wsb wa wi wal equtr2 ax12v syl impancom alrimiv df-sb sylibr ex
-      ) BCEZAABCFZRAGZDCEZBDEZAHBIZHZDISTUDDRUAAUCRUAGUBAUCHBDCJABDKLMNABDCOPQ
-      $.
+      ( vy weq wi wal wsb equeucl ax12v syl6 com23 alrimdv df-sb syl6ibr ) BCEZ
+      ADCEZBDEZAFBGZFZDGABCHPATDPQASPQRASFBDCIABDJKLMABDCNO $.
```

Comment:
This could have a "proof shortened" tag if you want.

Reply:
(no tag added)